GliaWidgets/Theme/Alert/ConfirmationAlertConfiguration.swift

```diff
@@ -1,8 +1,20 @@
+/// Configuration of a confirmation alert.
```

Comment:
```suggestion
/// Configuration of the confirmation alert.
```

Reply:
Not sure about that one. It's not THE specific confirmation alert, it's any of them. I trust your judgement, so I changed it to `the`, but this one is controversial IMO.